test/datastream/datastream_test.go

```diff
@@ -108,14 +122,40 @@ var _ = Describe("Streaming Data Conversion", func() {
 
 				By("Checking the output of the data stream")
 				Expect(err).NotTo(HaveOccurred(), "ioutil.ReadAll erred")
-				Expect(int64(output.Len())).To(Equal(size))
-				//Expect(output.Bytes()).To(Equal(sampleData)) // TODO replace with checksum?
+				if ff[0] == image.ExtQcow2 {
+					By("Checking getImageVirtualSize")
```

Comment:
what do you think about defining a bool in the test struct saying whether or not to use the virtualized size vs the `stat` file size, rather than hard-coding a format type?

Reply:
I like it